composer.json

```diff
@@ -22,17 +22,17 @@
         "league/oauth2-server": "^8.3",
         "nyholm/psr7": "^1.4",
         "psr/http-factory": "^1.0",
-        "symfony/event-dispatcher": "^5.4|^6.2",
-        "symfony/framework-bundle": "^5.4|^6.2",
+        "symfony/event-dispatcher": "^5.4|^6.2|^7.0",
+        "symfony/framework-bundle": "^5.4|^6.2|^7.0",
         "symfony/polyfill-php81": "^1.22",
         "symfony/psr-http-message-bridge": "^2.0|^6|^7",
-        "symfony/security-bundle": "^5.4|^6.2"
+        "symfony/security-bundle": "^5.4|^6.2|^7.0"
```

Comment:
Looks like some tests need to be fixed for lowest (5.4) and highest deps, I'm on it

Reply:
Thanks, I can take care of the 7.0 ones

Reply:
hmm for the 5.4 build, I suspect fixing them means trying to fix bugs that have been fixed in more recent patch releases. We may just want to remove the `--prefer-lowest` composer flag in the build setup

Reply:
So I've dropped it - for some reason most deps weren't fetched at all in this test (like doctrine bundle), I wasn't able to reproduce this issue.